src/misc/time-parametr-parser.ts

```diff
@@ -0,0 +1,43 @@
+export function parseTimeInput(args: string[]): string {
+  const allowedArgs = [
+    { argName: 'days', regex: /[0-9]{1,}/g },
+    { argName: 'hours', regex: /[0-9]{1,}/g },
+    { argName: 'ms', regex: /[0-9]{1,}/g },
+    { argName: 'date', regex: /[0-9]{4}-[0-9]{2}-[0-9]{2}/g },
+    {
+      argName: 'longdate',
+      regex: /[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}/g,
+    },
+  ];
+  const allowedArgsNames = allowedArgs.map((arg) => arg.argName);
+  let time: string = undefined;
+  args.forEach((inputArg) => {
```

Comment:
haven't you considered using [`minimist`](https://www.npmjs.com/package/minimist) or similar lib to handle input handling?